File: app/src/main/java/org/tesira/mturba/civichelper/card/CardColor.java
package org.tesira.mturba.civichelper.card;

/**
 * Each {@link Advance} is of one color with the exception of Mysticism, Written Record, Theocracy,
 * Literacy, Philosophy, Mathematics and Wonders of the World which belong to two {@link CardColor}.
 */
public enum CardColor {
    BLUE("Arts"),
    GREEN("Science"),
    ORANGE("Crafts"),
    RED("Civic"),
    YELLOW("Religion");

    private String name;

    CardColor(String name) {

        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/kotlin/org/tesira/civic/db/CardColor.kt
package org.tesira.civic.db

/**
 * Each [Card] is of one color with the exception of Mysticism, Written Record, Theocracy,
 * Literacy, Philosophy, Mathematics and Wonders of the World which belong to two [CardColor].
 */
enum class CardColor(val colorName: String) {
    BLUE("Arts"),
    GREEN("Science"),
    ORANGE("Crafts"),
    RED("Civic"),
    YELLOW("Religion")
}
